_posts/2018-04-16-the-hubris-of-man.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: The Hubris of Man
+date: 2018-04-16
+categories:
+    - SmashBros,
+    - Python,
+    - Flask,
+    - SQLAlchemy,
+    - PostgreSQL,
+    - HTML,
+    - Javascript,
+    - Jeffrey'sMajorProject
+description: Hubris is a Python Flask application meant as an introduction to Python Flask, PostgreSQL, and Web Development.  It is a joint project between Elijah Bendinsky and Jeffrey Taglic where Elijah Bendinsky created Smash Queue, a service that provides a basic queue implementation allowing users to keep track of the adds and drops during smash in the lounge.  Jeffrey Taglic is in charge of Tournaments, a service allowing users to create a tournament bracket, manage said bracket, and finally complete a major project
+image: 
+image-sm: 
+author: Elijah Bendinsky
+author-image: http://zedoax.me/img/logo.png
+author-bio: Screeeeeeee
```

Comment:
This is public facing, maybe make its something short about your year-level and major.